dist/source/repo-creator/sponsorship.html

```diff
@@ -0,0 +1,40 @@
+<template>
+	<require from="./choose-repository.css"></require>
+
+	<div class="container-fluid choose-repository">
+		<div class="row">
+			<div class="col-xs-12 panel-group" id="accordion">
```

Comment:
If this is going to be its own page, it probably shouldn't be an accordion.  Is there a plan for adding other things to this page?  There are other references to an accordion below that should be removed as well (and replaced as necessary with some other UI control).